samples/drivers/CAN/src/main.c

```diff
@@ -172,10 +176,12 @@ void led_thread(void *msgq, void *can_dev_param, void *gpio_dev_param)
 	}
 }
 
-void rx_button_isr(struct zcan_frame *msg)
+void rx_button_isr(struct zcan_frame *msg, void *arg)
 {
 	u16_t cnt = msg->data[0] | (msg->data[1] << 8);
```

Comment:
I usually change the position to ```u16_t cnt = (msg->data[1] << 8) | msg->data[0];``` It seems to be more natural.

Reply:
The sample needs to be updated anyways. I'll provide a new one after this gets merged.